docs/vars.md

```diff
@@ -164,6 +164,14 @@ node_taints:
   * `audit_policy_file`: "{{ kube_config_dir }}/audit-policy/apiserver-audit-policy.yaml"
 
   By default, the `audit_policy_file` contains [default rules](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/control-plane/templates/apiserver-audit-policy.yaml.j2) that can be overridden with the `audit_policy_custom_rules` variable.
+* *kubernetes_audit_webhook* - When set to `true`, enables the webhook audit backend.
+  The webhook parameters can be tuned via the following variables (which default values are shown below):
+  * `audit_webhook_config_file`: "{{ kube_config_dir }}/audit-policy/apiserver-audit-webhook-config.yaml"
+  * `audit_webhook_server_url`: `"https://audit.app"`
+  * `audit_webhook_server_extra_args`: {}
+  * `audit_webhook_mode`: batch
+  * `audit_webhook_batch_max_size`: 100
+  * `audit_webhook_batch_max_wait`: 1s
```

Comment:
This point is out of scope from this PR:
According to https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#batching audit_webhook_batch_max_size and audit_webhook_batch_max_wait are used when the audit_webhook_mode is batch.
So these options should be written to kubeadm-config.v1beta2.yaml if the mode is batch only.

Reply:
https://github.com/kubernetes-sigs/kubespray/pull/7444 is created for the above point.